blogs.yml

```diff
@@ -0,0 +1,253 @@
+# Please add your entries to a random position in the file.

```

Comment:
Unwanted file,  remove it pls.
Add it to `_data/blogs.yml`

Reply:
which is the unwanted file. The commit has only 2 files - blogs.yml and the new pic file.